docker-hub/scim.md

```diff
@@ -40,6 +40,8 @@ Before making SCIM configuration changes in your IdP, navigate to [Docker Hub](h
 
 ![scim-app-provisioning](images/scim-app-provisioning.png){:width="700px"}
 
+6. Remove the unwanted attributes from your mappings and fields from the Profile editor.
```

Comment:
Profile editor is used to modify custom user attributes, we don't need them to do that, they just need to remove them from the Attributes mappings
```suggestion
6. Remove all fields that are not supported from your Attributes Mappings.
```

And add this image a
![Untitled](https://user-images.githubusercontent.com/11965230/175658782-13cab083-9613-4c01-aa53-cc229f38751d.png)
s reference of the attributes we do support:

Reply:
Thank you, updated!